calico/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,11 @@
+name: Calico
+files:
+  - name: calico.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/openmetrics
+      - template: instances
+        options:
+          - template: instances/openmetrics
+            required: true
```

Comment:
```suggestion
```
required should be applied to a config option, it doesn't mean anything if applied to a template